src/windows/new_ride.c

```diff
@@ -290,7 +290,7 @@ static void window_new_ride_populate_list()
 
 	uint8 currentCategory = _window_new_ride_current_tab;
 	ride_list_item *nextListItem = (ride_list_item*)0x00F43523;
-	rct_ride_type **rideEntries = (rct_ride_type**)0x009ACFA4;
+	rct_ride_type **rideEntries = (rct_ride_type**)RCT2_ADDRESS_RIDE_ENTRIES;
```

Comment:
If there's no macro (yet), it means that we haven't figure out what it is (or haven't named it yet). I'd love to give them all macros, but I don't know if that's what people would want.